src/usr/bin/google_set_hostname

```diff
@@ -15,6 +15,25 @@
 
 # Deal with a new hostname assignment.
 
+retain_legacy_behavior() {
```

Comment:
please add a comment line describing the intended behavior of this routine. I.e what is the definition of legacy behavior and what it means to retain it here.

Reply:
Done.